editor/components/block-mover/style.scss

```diff
@@ -17,14 +19,21 @@
 	padding: 0;
 	width: $block-side-ui-width;
 	height: $block-side-ui-width;	// the side UI can be no taller than 2 * $block-side-ui-width, which matches the height of a line of text
+	border-radius: $button-style__radius-roundrect;
 
 	&[aria-disabled="true"] {
 		cursor: default;
 		color: $light-gray-300;
 		pointer-events: none;
 	}
 
-	// Try a background, only for nested situations @todo
+	svg {
+		width: $block-side-ui-width;
+		height: $block-side-ui-width;
+		padding: ( $block-side-ui-width - 18px ) / 2;	// this makes the SVG fill the whole available area, without scaling the artwork
+	}
+
+	// Apply a background in nested contexts, only on desktop
```

Comment:
In nested contexts on mobile the contrast may be low:
![image](https://user-images.githubusercontent.com/11271197/38870696-87c8c4be-4246-11e8-869b-3ad7736fdb50.png). But it seems like the result is better than what we had and finding a solution for this cases is really hard has the background color of the nested context is impossible to predict.